docs/dev_guides/changing_splink/contributing_to_docs.md

```diff
@@ -16,21 +16,27 @@ Once you've finished updating Splink documentation we ask that you run our spell
 
 ## Spellchecking docs
 
-When updating Splink documentation, we ask that you run our spellchecker before submitting a pull request. This is to help ensure quality and consistency across the documentation. Please note, the spellchecker _only works on markdown files_ and currently only works on systems which support `Homebrew` package manager. Instructions for other operating systems will be released later.
+When updating Splink documentation, we ask that you run our spellchecker before submitting a pull request. This is to help ensure quality and consistency across the documentation. Please note, the spellchecker _only works on markdown files_.
+
+If you are a Mac user with the `Homebrew` package manager installed, the script below will automatically install
+the required system dependency, `aspell`.
+If you've created your development environment [using conda](./development_quickstart.md), `aspell` will have been installed as part of that
+process.
+Instructions for installing `aspell` through other means may be added here in the future.
 
 To run the spellchecker on either a single markdown file or folder of markdown files, you can use the following script:
 
 ```sh
-source scripts/pyspelling/spellchecker.sh <path_to_file_or_folder>
+./scripts/pyspelling/spellchecker.sh <path_to_file_or_folder>
```

Comment:
Just to note this now - we may want to make a note that the file may need to be executable: `chmod -x`.

Reply:
We could do this, but Git stores the file permissions, so anyone who gets it from Git should generally not need to do this.

Reply:
I did not know that, interesting!

Reply:
My issue with the `./` is that it requires the current directory to be the top-level Splink-1 (or equiv) directory. We could just include the location of the file to run in the code block in the docs for easy copy/paste, together with some instructions, e.g.:

> To run the spellchecker on either a single markdown file or folder of markdown files, call the following script in your terminal:
> 
> ```sh
> scripts/pyspelling/spellchecker.sh <path_to_file_or_folder>
> ```
> This cal be done by placing `source`, etc ...

Reply:
`source` isn't any different from `./` in this way (and they differ in other ways, such that this script should not be run with `source`, I think). It seems fine to imply that they should be in the splink directory, but if we don't want to leave that implied we should just say it.

Reply:
Agreed, so I think just the file path is best, with or without `splink` at the start - happy for you to go with what you think is most sensible

Reply:
@zmbc are you happy to remove the `./` at the start of the location of the bash script? This way we can generically point to the script without worrying about whether users can use `source` or are in the correct location to ensure `./` works

Reply:
`./` doesn't have any effect on those issues -- it's completely equivalent to leaving it out. That _is_ using the file path as you mentioned above, so I thought we had agreed this was okay as-is. If you feel strongly I can remove it.

Reply:
Apologies for the confusion - we did agree, I did think it would be clearer instruction-wise without the `.`, but this is a minor issue so happy to leave it and merge the PR.